source/vistas/core/gis/elevation.py

```diff
@@ -214,3 +225,31 @@ def measure(lat1, lon1, lat2, lon2):
         task.status = task.INDETERMINATE
         new_plugin.calculate_stats()
         return new_plugin
+
+    def create_data_dem(self, extent, zoom, merge=False):
+        """
+        Creates an elevation grid from elevation data in memory.
+        :param extent Extent to build the DEM for.
+        :param zoom The zoom level to render the DEM at.
+        :param merge Indicate whether to return the DEM as a single numpy grid, or as a cache, using mercantile.Tile
+        objects to index into the cache.
+        :return {mercantile.Tile: numpy.ndarray} if merge==False, numpy.ndarray if merge==True
```

Comment:
It's somewhat ambiguous in terms of what it returns. This code should probably be refactored to return a single `numpy.ndarray`, and have a slightly different function that does similar stuff, but returns the individual tile grids. Right now, the `merge` keyword is the only difference.

Very useful. Somewhat confusing right now, admittedly.